README.md

```diff
@@ -21,6 +21,9 @@ for machine learning. It features an imperative, *define-by-run* style user API.
 *define-by-run* API, the code written with Optuna enjoys high modularity, and the user of
 Optuna can dynamically construct the search spaces for the hyperparameters.
 
+## News
+
+- **2020-08-07** We are welcoming [contributions](#contribution) and are working on streamlining the experience
```

Comment:
[Just suggestion]

How about putting the link to [medium post](https://medium.com/optuna/optuna-wants-your-pull-request-ff619572302c)? I think it is a good starting point and describes the process of contribution in detail.

Reply:
That's an idea as well. One of purposes of this PR was to make the new contribution section in the README more visible. Did you mean to add a second link in this bullet?

Reply:
> add a second link

Yes, I intended to add a second link. But please let me know if you think it's enough to show a link to the contribution section. (Then I'll merge after your response.) :pray:

Reply:
Thanks for the clarification, I added a link. It's a good thing to spread our Medium blog as well.